docs/source/cluster_management/clis.rst

```diff
@@ -45,4 +45,20 @@ timestamp
 
 Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`` for more information.
 
+Offline CLIs
+============
+
+Due to their destructive nature for in-progress transactions, there are two sub-commands of the timestamp CLI which can only be run when AtlasDB is offline. These are:
+
+  - ``clean-transactions``
+  - ``fast-forward``
+
+To run these CLIs, first ensure that all of your AtlasDB clients are all shut down, and then run the CLI with the ``--offline`` flag.
+
+For example, to run the ``fast-forward`` command with default settings, run ``./bin/atlasdb --offline timestamp fast-forward``.
+
+To add custom configuration, use ``./bin/atlasdb --offline timestamp fast-forward --config <path/to/atlasConfig.yaml>``.
```

Comment:
It's not really a custom configuration. When running the CLIs manually (so not part of dropwizard), it's required to list the config. I don't think this command will work since `--config` is in the wrong place.